recipe/meta.yaml

```diff
@@ -25,26 +28,29 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
   host:
-    - python >=3.6,<4
+    - python
     - pip
     - cython
     - setuptools >18.0
     - setuptools_scm >1.5.4
     - wheel
   run:
-    - python >=3.6,<4
+    - python
     - numpy >=1.7
     - msgpack-python
 
 test:
   requires:
     - pytest
+    - python
```

Comment:
Do we need python in the test environment? It's already in `run` so it will be in the test environment too